packages/fiori/src/Timeline.js

```diff
@@ -96,6 +114,18 @@ class Timeline extends UI5Element {
 
 		this._itemNavigation.setCurrentItem(target);
 	}
+
+	onAfterRendering() {
+		this._itemNavigation.NavigationMode = this.axisOrientation === AXIS_ORIENTATION.horizontal ? NavigationMode.Horizontal : NavigationMode.Vertical;
```

Comment:
Here you can use the defined type for the enum

Reply:
Done